source/_integrations/yolink.markdown

```diff
@@ -16,31 +16,10 @@ ha_integration_type: integration
 
 Integrates [YoLink](https://www.yosmart.com/) Devices into Home Assistant.
 
-You must contact <service@yosmart.com> to obtain a `client_id` and `client_secret`.
-
-To enable this platform, add the following lines to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-yolink:
-  client_id: CLIENT_ID
-  client_secret: CLIENT_SECRET
-```
-
-Restart Home Assistant. Then, go to the frontend and authorize YoLink.
-
 {% include integrations/config_flow.md %}
 
-{% configuration %}
-client_id:
-  description: Your YoLink API client ID.
-  required: true
-  type: string
-client_secret:
-  description: Your YoLink API client secret.
-  required: true
-  type: string
-{% endconfiguration %}
+Unless configured otherwise, Home Assistant will use account linking provided by
+Nabu Casa for authenticating with YoLink, If you want to use separate credentials, Please contact <service@yosmart.com> to obtain a `client_id` and `client_secret`, Then you can add your credentials via application credentials.
```

Comment:
```suggestion
Nabu Casa for authenticating with YoLink. If you want to use separate credentials, please contact <service@yosmart.com> to obtain a `client_id` and `client_secret`. Then you can add your credentials via application credentials.
```

Reply:
Maybe we need to say where to find the application credentials setting? @frenck what do you think?